testdata/bootstrap/client.yaml.tmpl

```diff
@@ -60,7 +60,7 @@ static_resources:
           metadata:
             filter_metadata:
               istio:
-                workload: ratings-v1;default;ratings;version-1
+                workload: ratings-v1;default;ratings;version-1;test-cluster
```

Comment:
nit: would be good to have a different value for destination cluster than source.

Reply:
Done.